src/style.css

```diff
@@ -1 +1,43 @@
-/* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
There is no need to set these properties globally, a better approach is to apply flexbox to specific containers or components.